wrappers/python/nntile/inference/api_server_base.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+
+import uvicorn
+
+
+@dataclass
+class SimpleApiServerParams:
+    host: str = "127.0.0.1"
+    port: int = 12224
+
+
+class SimpleApiServerBase(ABC):
```

Comment:
I totally don't get it. It sees to bee excessively overengineered. Why do you need this class hierarchy? What is API exactly? Just define two handlers `/liveness` for liveness probe and `/complete` for text completion in `bottle`, that's all.

Reply:
The logic here is:
1. We will have several different models with different apis. Like now gpt2 only and Unet++ is coming
2. I can't do one api for all just because, for example, image models and llm have different api
3. I don't want devs to copy past any code for server or develop bycicles.
4. So that is it: 
if you want to do new inference server - just write your class with get_app() returning fastapi app. And base class will take care all the things

That do you think of this logic?

Reply:
What is the goal? Is it about a new fancy library for building general inference services or just inference facade for nntile?

Reply:
Just inference facade. Later with some optimization for generation (like you see in vllm/deepspeed inference - qvcache and smart batching)

For now, I don't really understand the point of overengineering claim. Just small base class with prebuild logic

FYI: Maybe it's an excellent idea to move it to a different repo at some point. But it's further ahead

Reply:
resolving this, as it becomes subjective. I am just doing what a team decided, it is not my fancy thoughts
If you believe I am wrong - please go to our meeting. Or just ask to organize in person/zoom

Reply:
> For now, I don't really understand the point of overengineering claim. Just small base class with prebuild logic

Point is KEEP IT SIMPLE. You DO NOT NEED base classes and inheritance until you ACTUALLY NEED IT.

Reply:
It is simple enough. I describe why I did this way